src/app/charts/line-graph.component.ts

```diff
@@ -79,7 +79,7 @@ export class LineGraphComponent implements OnChanges, AfterContentInit {
     }
 
     /* Executes on every @Input change */
-    ngOnChanges(): void {
+    ngOnChanges(changes?: any): void {
```

Comment:
Was this change necessary for some reason? I don't think we need to redeclare the type of this method.

Reply:
Leftover from playing around, good catch